packages/core/src/Username.ts

```diff
@@ -0,0 +1,11 @@
+import * as t from 'io-ts';
+
+interface UsernameBrand {
+  readonly Username: unique symbol;
+}
+
+export type Username = t.Branded<string, UsernameBrand>;
+
+const isValid = (v: string): v is Username => v.length > 5 && v.length < 25;
```

Comment:
It's better to have a more strict guard on the username.
We must forbid weird characters.